Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -1,58 +1,26 @@
 const SteamScript = include("engines.wine.quick_script.steam_script");
-
-const DotNET472 = include("engines.wine.verbs.dotnet472");
-const Vcrun2017 = include("engines.wine.verbs.vcrun2017");
+const dotnet472 = include("engines.wine.verbs.dotnet472");
+const vcrun2017 = include("engines.wine.verbs.vcrun2017");
```

Comment:
Is there a reason for this rename?

Reply:
@plata suggested me to use more standarized names for includes